docs/server/source/quickstart.md

```diff
@@ -8,7 +8,7 @@ A. Install MongoDB as the database backend. (There are other options but you can
 
 B. Run MongoDB. Open a Terminal and run the command:
 ```text
-$ mongod --replSet=bigchain-rs
+$ sudo mongod --replSet=bigchain-rs
```

Comment:
Today if you take the Quickstart page literally and just do `mongod --replSet=bigchain-rs` then you will get an error like "exception in initAndListen: 20 Attempted to create a lock file on a read-only directory: /data/db, terminating". That's not a great user experience.

@sbellem suggested two other fixes which would also work. We should just tell them to do one though; it's a Quickstart page, not an exhaustive list of all possible ways to get going with BigchainDB.